lib/l10n/app_fr.arb

```diff
@@ -255,285 +249,339 @@
-  "@disableGesture": {},
-  "removeFromPlaylistTooltip": "Retirer de la playlist",
-  "@removeFromPlaylistTooltip": {},
-  "removeFromPlaylistTitle": "Retirer de la playlist",
-  "@removeFromPlaylistTitle": {},
-  "removedFromPlaylist": "Retirer de la playlist.",
-  "@removedFromPlaylist": {},
-  "disableGestureSubtitle": "Option permettant de désactiver les gestes.",
-  "@disableGestureSubtitle": {},
-  "showUncensoredLogMessage": "Ce journal contient vos informations de connexion. L'afficher ?",
-  "@showUncensoredLogMessage": {},
-  "insertedIntoQueue": "Ajouter à la file d'attente.",
-  "@insertedIntoQueue": {
-    "description": "Snackbar message that shows when the user successfully inserts items into the play queue at a location that is not necessarily the end."
-  },
-  "refresh": "ACTUALISER",
-  "@refresh": {},
-  "confirm": "Confirmer",
-  "@confirm": {},
   "language": "Langue",
-  "@language": {},
-  "playNext": "Lire à la suite",
-  "@playNext": {
-    "description": "Popup menu item title for inserting an item into the play queue after the currently-playing item."
-  },
+  "confirm": "Confirmé",
+  "showUncensoredLogMessage": "Ce journal contient vos informations de connexion. Afficher tout de même ?",
   "resetTabs": "Réinitialiser les onglets",
-  "@resetTabs": {},
-  "noMusicLibrariesBody": "Finamp n'a trouvé aucune bibliothèque musicale. Assurez-vous que votre serveur Jellyfin contient au moins une médiathèque de type \"Musique\".",
-  "@noMusicLibrariesBody": {},
-  "noMusicLibrariesTitle": "Pas de bibliothèque musicale",
+  "noMusicLibrariesTitle": "Aucune bibliothèque musicale",
   "@noMusicLibrariesTitle": {
-    "description": "Title for message that shows on the views screen when no music libraries could be found."
+    "description": "Titre du message affiché sur l'écran de vues lorsqu'aucune bibliothèque musicale n'a été trouvée."
   },
-  "deleteDownloadsPrompt": "Voulez-vous vraiment supprimer {itemType, select, album{album} playlist{playlist} artist{artist} genre{genre} track{song} other{}} '{itemName}' de cet appareil ?",
-  "@deleteDownloadsPrompt": {
+  "noMusicLibrariesBody": "Finamp n'a pas trouvé de bibliothèques musicales. Veuillez vous assurer que votre serveur Jellyfin contient au moins une bibliothèque avec le type de contenu défini sur \"Musique\".",
+  "refresh": "RAFRAÎCHIR",
+  "swipeInsertQueueNext": "Lire la chanson balayée en prochain",
+  "@swipeInsertQueueNext": {},
+  "swipeInsertQueueNextSubtitle": "Activer l'insertion d'une chanson comme élément suivant dans la file d'attente lorsqu'elle est balayée dans la liste des chansons, au lieu de l'ajouter à la fin.",
+  "@swipeInsertQueueNextSubtitle": {},
+  "redesignBeta": "Essayer la bêta",
+  "@redesignBeta": {},
+  "playbackOrderShuffledTooltip": "Lecture en aléatoire. Touchez pour activer/désactiver.",
+  "@playbackOrderShuffledTooltip": {},
+  "playbackOrderLinearTooltip": "Lecture du l'ordre. Tap to toggle.",
```

Comment:
I don't get this line

Reply:
En français y'a vraiment une difficulté concernant l'aléatoire, je n'aime pas vraiment utilisé les termes "mélanger" car en soit on parle juste de "lecture aléatoire", cette ligne si j'ai bien compris, est censé lire dans l'ordre, je m'étais trompé dans l'écriture, je pense changer comme ça dans mon commit de correction à venir :
```suggestion
  "playbackOrderLinearTooltip": "Lecture dans l'ordre. Touchez pour activer/désactiver.",
```

Reply:
À voir dans son contexte effectivement, mais ça me semble bon 👌